.github/workflows/release_helm.yml

```diff
@@ -2,8 +2,10 @@ name: Build, test and publish Helm chart
 
 on:
   push:
-    tags:
-    - 'v*'
+    branches:
+    - 'master'
+    paths:
+    - "charts/yet-another-cloudwatch-exporter/Chart.yaml"
```

Comment:
On a second thought, might be better to leave this as it was before.

Merging this PR would cause a new helm release for yace `v0.43.0-alpha`, but we've merged a few PRs already since `v0.43.0-alpha`.

When compiling the changelog for the next release, I'll make sure that the chart is updated as well (we can write a quick CI check for that as well).

Reply:
Ok 😄 

Quick question: on which kind of modification would like to release the helm chart ?

Reply:
Ok 👍 
It's done in #763 
Last question regarding that new PR do you want me to keep the current behavior where the pipeline is triggered on git tag ?